src/lib.rs

```diff
@@ -293,9 +293,7 @@ pub fn generate_flamegraph_for_workload(workload: Workload, opts: Options) -> an
 
     #[cfg(target_os = "linux")]
     {
-        if let Some(skip_after) = &opts.flamegraph_options.skip_after {
-            collapse_options.skip_after = Some(skip_after.into())
-        }
+        collapse_options.skip_after = opts.flamegraph_options.skip_after.clone();
```

Comment:
Why do we need to clone here?

Reply:
Because a few lines below, `into_inferno()` fully consumes `flamegraph_options`, despite not being interested in the `.skip_after` field.

I think it was doing a copy before already. We could probably drain/reassign the vector to make it clear it isn't going to be used again, but it also seems wrong to modify the command-line options.